extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -557,12 +562,16 @@ export class AmpSlideScroll extends BaseSlides {
       // Set the correct direction.
       direction = direction < 0 ? 1 : -1;
     }
-    this.analyticsEvent_('amp-carousel-change');
+    const vars = {
+      'fromSlide': this.dataSlideIdArr[this.slideIndex_],
+      'toSlide': this.dataSlideIdArr[newSlideIndex],
+    };
+    this.analyticsEvent_('amp-carousel-change', vars);
     // At this point direction can be only +1 or -1.
     if (direction == 1) {
-      this.analyticsEvent_('amp-carousel-next');
+      this.analyticsEvent_('amp-carousel-next', vars);
```

Comment:
Do we need it detailed like this, or could it be abstract like `content-view` with just the info for the newly viewed content. Then it could be used for other stuff like accordions, tabs.